wire-compiler/src/main/java/com/squareup/wire/schema/Target.kt

```diff
@@ -261,11 +261,12 @@ data class KotlinTarget(
 }
 
 data class ProtoTarget(
-  override val includes: List<String> = listOf(),
-  override val excludes: List<String> = listOf(),
-  override val exclusive: Boolean = true,
   val outDirectory: String
 ) : Target() {
+  override val includes: List<String> = listOf()
```

Comment:
This change is unrelated to `ClaimedDefinitions`, wanna move it to a separate PR?

Reply:
yeah removed it